### DISCUSSION TITLE
browser.mock() support on AWS Code build

### DISCUSSION THREAD

**Discussion by parida1611** [webdriverio/webdriverio#12441] [Q&A]
Hi Team, I want to use browser.mock() command for one of my UI automation project. But the fact that this command can not be used when running automated tests in the cloud is concerning to me as very soon my automation execution will be migrating to AWS Code Build. 

As cloud is a very broad term, I want to know if this will work on AWS Code Build or not where we'll still be using the chrome browser capabilities in headless mode but the execution will be happening on the AWS Code Build. Do we know that if this will work or not in this scenario?

Thanks and waiting for response.

**Comment by christian-bromann** [comment]
@parida1611 I think the wording is a bit misguiding here. The command can not be used if a test is executed separately from where the browser is automated. For example this command can't be used when running tests on vendors like Sauce Labs or BrowserStack since they don't allow us to connect to the browser in way we need to enable this command.

Since on AWS Cloud Build you run the browser next to the test, it should work. Please try it out and let me know if you experience any issues.
